htmresearch/algorithms/faulty_temporal_memory.py

```diff
@@ -58,6 +49,7 @@ def killCells(self, percent = 0.05):
     time you call this method a permutation list is set up. Calls change the
     number of cells considered dead.
     """
+
```

Comment:
With the code as-is, `activatePredictedColumns` will sometimes activate dead cells. (`burstColumns` will do the right thing.)